packages/sources/dlc-btc-por/src/config/index.ts

```diff
@@ -0,0 +1,56 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig(
+  {
+    RPC_URL: {
+      description: 'The RPC URL to connect to the EVM chain',
+      type: 'string',
+      required: true,
+    },
+    CHAIN_ID: {
+      description: 'The EVM chain id to connect to',
+      type: 'number',
+      required: true,
+      default: 42161,
+    },
+    DLC_CONTRACT: {
```

Comment:
similar comments as the README, I don't think there's anything wrong with this, but do you think we it would be worth making these configurable at the input param level?